### ISSUE TITLE
[explicit-function-return-type] Unfortunate typo in documentation

### ISSUE THREAD

**Issue by onalant** [typescript-eslint/typescript-eslint#769]
In the last example under [allowTypedFunctionExpressions](https://github.com/typescript-eslint/typescript-eslint/blob/master/packages/eslint-plugin/docs/rules/explicit-function-return-type.md#allowtypedfunctionexpressions), `functionWithObjectArg` is written as
```typescript
declare functionWithObjectArg(arg: { meth: () => number });
functionWithObjectArg({
  meth() {
    return 1;
  },
});
```
rather than what I imagine was the intended code:
```typescript
declare functionWithObjectArg(arg: { method: () => number });
functionWithObjectArg({
  method() {
    return 1;
  },
});
```
My apologies if I have mislabeled this issue.

**Comment by bradzacher** [comment]
a bit of lazy shorthand, yeah.
feel free to PR change it.

For future reference, making single-file changes is easy - you should be able to edit the file within the github UI, and it will automatically let you create a fork and raise a PR. It only takes a few clicks.

**Comment by onalant** [comment]
Will do. I was not aware of the fact that UI-made changes create a PR. Thanks for the pointer!

Kimden: Brad Zacher<mailto:notifications@github.com>
Gönderilme: 28 Temmuz 2019 Pazar 20:28
Kime: typescript-eslint/typescript-eslint<mailto:typescript-eslint@noreply.github.com>
Bilgi: Tarik Onalan<mailto:onalante@outlook.com>; Author<mailto:author@noreply.github.com>
Konu: Re: [typescript-eslint/typescript-eslint] [explicit-function-return-type] Unfortunate typo in documentation (#769)


a bit of lazy shorthand, yeah.
feel free to PR change it.

For future reference, making single-file changes is easy - you should be able to edit the file within the github UI, and it will automatically let you create a fork and raise a PR. It only takes a few clicks.
